edit/rbac.authorization.k8s.io.roletemplate.vue

```diff
@@ -0,0 +1,327 @@
+
+      return verbs.includes(verb);
+    },
+
+    setVerb(verb, value, rule) {
+      this.$set(rule.verbs, rule.verbs || []);
+
+      if (value) {
+        const index = rule.verbs.indexOf(verb);
+
+        if (index === -1) {
+          rule.verbs.push(verb);
+        }
+      } else {
+        const index = rule.verbs.indexOf(verb);
+
+        if (index !== -1) {
+          rule.verbs.splice(index, 1);
+        }
+      }
+    },
+    setRule(key, rule, value) {
+      this.$set(rule, key, [value]);
+    },
+    getRule(key, rule) {
+      return rule[key]?.[0] || null;
+    }
+  }
+};
+</script>
+
+<template>
+  <CruResource
+    class="receiver"
+    :can-yaml="!isCreate"
+    :done-route="doneRoute"
+    :mode="mode"
+    :resource="value"
+    :selected-subtype="value.subtype"
+    :subtypes="subtypes"
+    :errors="errors"
+    @select-type="selectType"
+    @error="e=>errors = e"
+    @finish="save"
+    @cancel="done"
+  >
+    <NameNsDescription
+      v-model="value"
+      :namespaced="false"
+      :mode="mode"
+      label="Name"
+    />
+    <div class="row">
+      <div v-if="isRancherSubtype" class="col span-6">
+        <RadioGroup
+          v-model="value.locked"
+          name="storageSource"
+          :label="t('rbac.roletemplate.locked.label')"
+          class="mb-10"
+          :options="lockedOptions"
+        />
+      </div>
+      <div class="col span-6">
+        <RadioGroup
+          :value="value.default"
+          name="storageSource"
+          :label="defaultLabel"
+          class="mb-10"
+          :options="newUserDefaultOptions"
+          @input="value.updateDefault"
+        />
+      </div>
+    </div>
+    <div class="spacer"></div>
+    <Tabbed :side-tabs="true">
+      <Tab
+        name="grant-resources"
+        label="Grant Resources"
+        :weight="1"
+      >
+        <ArrayList
+          v-model="value.rules"
+          label="Resources"
+          :default-add-value="defaultRule"
+          :initial-empty-row="true"
+          :show-header="true"
+          add-label="Add Resource"
+        >
+          <template #column-headers>
+            <div class="column-headers row">
+              <div class="col span-3">
+                <label class="text-label">Verbs</label>
+              </div>
+              <div class="col span-3">
+                <label class="text-label">Resources</label>
+              </div>
+              <div class="col span-3">
+                <label class="text-label">Non-Resource URLS</label>
+              </div>
+              <div class="col span-3">
+                <label class="text-label">API Groups</label>
+              </div>
+            </div>
+          </template>
+          <template #columns="props">
+            <div class="columns row">
+              <div class="col span-3">
+                <Select
+                  v-model="props.row.value.verbs"
+                  class="lg"
+                  :taggable="false"

```

Comment:
Should be taggable; Other verbs exist and you should be able to enter them.

Reply:
Done